tools/tap-driver

```diff
@@ -283,6 +296,9 @@ class TapDriver(Driver):
             self.trs.write(":test-global-result: FAIL\n")
             self.trs.write(":recheck: yes\n")
         elif skipped:
+            if self.skip_all_reason is not None:
+                self.result_skip("skipping due to: ", self.skip_all_reason)
```

Comment:
Why does this need to happen here again? Shouldn't the first check at l255 take care of this?

Reply:
Oops.  I tried two different approaches to this, clearly.  I'll delete this one.